package.json

```diff
@@ -129,7 +130,8 @@
         "ts-loader": "^6.2.2",
         "ts-node": "~10.8.1",
         "typescript": "~4.3.5",
-        "wait-on": "^6.0.1"
+        "wait-on": "^6.0.1",
+        "webpack": "^5.74.0"
```

Comment:
Why do we need to manually introduce webpack to deps?

Reply:
fixed